app/src/styles.sass

```diff
@@ -196,3 +196,11 @@ div.mat-menu-panel.nav-menu
   button.mat-menu-item:hover
     background-color: $nav-hover-bg
     color: white
+
+.mat-menu-content hr
+  background-color: $maincolor
+  border: none
+  height: 1px
+  width: 78%
+  @media (min-width: $breakpoint-tablet)
+    display: none
```

Comment:
This must not be here, in the global style file. We do want all the hr in all menus to look the same. But we do not want all the hr in all menus to disappear when the screen is wide. This has to be moved in navtitle.component.sass.